lib/CustomFields/components/CustomFieldsForm/CustomFieldsForm.js

```diff
@@ -0,0 +1,341 @@
+import React, { Fragment, useReducer, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import {
+  isEqual,
+  uniqueId,
+} from 'lodash';
+
+import {
+  AccordionSet,
+  ExpandAllButton,
+  Paneset,
+  Pane,
+  Button,
+  PaneMenu,
+  PaneHeaderIconButton,
+  PaneFooter,
+  TextField,
+  Col,
+  Row,
+} from '@folio/stripes-components';
+
+import NavigationModal from '../../../Notes/components/NavigationModal';
+
+import {
+  AddButton,
+  FieldAccordion,
+  TextboxFields,
+  DeleteModal,
+} from '..';
+
+import {
+  fieldTypes,
+  fieldTypesLabels,
+  defaultFieldConfigs,
+} from '../../constants';
+
+import styles from './CustomFieldsForm.css';
+
+const propTypes = {
+  entityType: PropTypes.element.isRequired,
+  fetchUsageStatistics: PropTypes.func.isRequired,
+  initialValues: PropTypes.arrayOf(PropTypes.shape({
+
```

Comment:
It might be better to use `propTypes.object` or describe this  `propTypes.shape`